stubs/Markdown/markdown/inlinepatterns.pyi

```diff
@@ -6,7 +6,7 @@ from xml.etree.ElementTree import Element
 from markdown import util
 from markdown.core import Markdown
 
-def build_inlinepatterns(md: Markdown, **kwargs) -> util.Registry[InlineProcessor]: ...
+def build_inlinepatterns(md: Markdown, **kwargs) -> util.Registry[Pattern]: ...
```

Comment:
InlineProcessor is a subclass of Pattern and everywhere else the more general class is correct. However this particular method really does create a registry of only InlineProcessor. If the covariance stuff all works out, I believe we should leave this part as is. I'll try later today what mypy thinks about it with the full codebase

```suggestion
def build_inlinepatterns(md: Markdown, **kwargs) -> util.Registry[InlineProcessor]: ...
```

Reply:
A `Registry` can’t be covariant because it’s mutable. That is, we can’t convert `reg1: Registry[InlineProcessor]` to `reg2: Registry[Pattern]` because then one could add a `Pattern` to `reg2` and read it out from `reg1` as an `InlineProcessor` even if it isn’t.

So if this is going to be used as `Registry[Pattern]`, it needs to be declared that way from the start.